HTAN.model.csv

```diff
@@ -1030,3 +1034,8 @@ Barretts Esophagus Goblet Cells Present,Presence or absennce of Barretts esophag
 Pancreatitis Onset Year,Date of onset of pancreatitis.,,,,FALSE,Follow Up,,,num
 HTAN Parent Channel Metadata ID,HTAN ID for a level 3 channels table.,,,,TRUE, Imaging Level 4,,,
 Single Nucleus Capture,Nuclei isolation method,"Plates, 10x, droplet",,,FALSE,scmC-seq Level 1,,,
+Associated mRNA Library Data File ID,Sample Level HTAN Data File ID for the associated level - HTAN ID of this file HTAN ID SOP (eg HTANx_yyy_zzz),,,,TRUE,Multiplexed CITE-seq Level 1,,,

```

Comment:
@Gibbsdavidl I'm a little confused by this attribute, with which template would this ID be originally annotated?

Reply:
If I understand the question, initially this attribute is annotated at level 1, but the submitter requested it also be available in all subsequent metadata tables.